pages/contribution-flow.js

```diff
@@ -130,8 +130,9 @@ class NewContributionFlowPage extends React.Component {
   }
 
   renderPageContent() {
-    const { data = {}, step, LoggedInUser, error } = this.props;
+    const { data = {}, step, paymentMethod, LoggedInUser, error } = this.props;
```

Comment:
By the way, this param/variable name is conflicting with other concepts and is confusing.

Reply:
Sure we will rename this. I think this I'll do it in another PR perhaps as it involves the whole file; we have a prop name called `paymentMethod`. 🤔

Reply:
@znarf : How does `donationMethod` sounds? Does that seem better to you? 🤔

Reply:
Done at https://github.com/opencollective/opencollective-frontend/pull/6953